ucte/ucte-network/src/main/java/com/powsybl/ucte/network/UcteValidation.java

```diff
@@ -0,0 +1,96 @@
+/**
+ * Copyright (c) 2020, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.ucte.network;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * @author Anne Tilloy {@literal <anne.tilloy at rte-france.com>}
+ */
+public final class UcteValidation {
```

Comment:
From my point of view it's strange to have two different classes for UCTE validation. I'm pretty sure you can implement all the checks directly on the UCTE model

Reply:
I jhave removed the check made on buses because if has to be done after a power flow calculation and so at another level than the import and the export. But for targetV checks (generators and transformers), it is really easier to make them on IIDM. On UCTE model, it is not easy to retrieve the voltage level from the generator and the regulating terminal of a transformer. Is it really an issue?